Gems/AzQtComponentsForPython/Code/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+      cmake_policy(SET CMP0071 NEW)
+    endif()
+
+    project(AzQtComponent_Bindings LANGUAGES CXX)
+    set(CMAKE_AUTOMOC ON)
+    find_package(Qt5 5.12 REQUIRED COMPONENTS Core Gui Widgets)
+                 
+    # The library we're creating bindings for.
+    set(AZQTCOMPONENTS_LIBRARY "AzQtComponents")
+    
+    # Load the library/exe locations. These are platform specific.
+    include(${pal_dir}/azqtcomponentsforpython_file_settings.cmake)
+    
+    #Define the shared library for which we will create bindings.
+    set_property(TARGET ${AZQTCOMPONENTS_LIBRARY} PROPERTY PREFIX "")
+
+    # Ensure the generated wrapper files have the correct header.
+    set(LICENSE_FILE ${CMAKE_CURRENT_LIST_DIR}/license_header.txt)
+
+    # Set up the options to pass to shiboken.
+    set(shiboken_options --generator-set=shiboken --enable-parent-ctor-heuristic
+        --enable-pyside-extensions --enable-return-value-heuristic --use-isnull-as-nb_nonzero
+        --avoid-protected-hack --language-level=c++17 --debug-level=full
+        --license-file=${LICENSE_FILE}
+        -I${LY_ROOT_FOLDER}/Code/Framework/AzCore
+        -I${LY_ROOT_FOLDER}/Code/Framework/AzQtComponents
+        -I${LY_ROOT_FOLDER}/Code/Framework/AzCore/Platform/${PAL_PLATFORM_NAME} 
+        -I${QT_MODULE_DIR}/include
+        -I${QT_MODULE_DIR}/include/QtWidgets
+        -I${QT_MODULE_DIR}/include/QtCore
+        -I${QT_MODULE_DIR}/include/QtGui
+        -T${PYSIDE2_TYPESYSTEM_PATH}
+        -T${PYSIDE2_SHARE_PATH}
+        --output-directory=${CMAKE_CURRENT_BINARY_DIR}
+    )
+
+    set(generated_sources_dependencies ${wrapped_header} ${typesystem_file})
+
+    add_custom_command(
+        OUTPUT ${WRAPPED_FILES}
+        COMMAND ${SHIBOKEN_GENERATOR_EXE_PATH} ${shiboken_options} ${wrapped_header} ${typesystem_file}
+        DEPENDS ${generated_sources_dependencies} 
+        COMMENT "Running generator for ${typesystem_file}."
+        VERBATIM
+    )
+
+    # Set up the project for building the code generated by shiboken.
+    ly_add_target(
+        NAME ${GEM_NAME}.Editor GEM_MODULE
+        NAMESPACE Gem
+        OUTPUT_NAME ${BINDINGS_LIBRARY}
+        FILES_CMAKE
+            wrapped_files.cmake
+        INCLUDE_DIRECTORIES
+            PRIVATE
+                ${PYSIDE2_INCLUDE_DIR}
+                ${PYTHON_INCLUDE_DIR}
+                ${SHIBOKEN_INCLUDE_DIR}
+                ${PYSIDE2_INCLUDE_DIR}/QtCore
+                ${PYSIDE2_INCLUDE_DIR}/QtGui
+                ${PYSIDE2_INCLUDE_DIR}/QtWidgets
+        BUILD_DEPENDENCIES
+            PUBLIC
+                3rdParty::pyside2
+                Qt5::Widgets
+                Qt5::Core
+                Qt5::Widgets
+                Qt5::Gui
+                ${AZQTCOMPONENTS_LIBRARY}
+                Gem::EditorPythonBindings.Static
+    )
+
+    ly_create_alias(NAME ${MODULE_NAME}.Builders NAMESPACE Gem TARGETS ${MODULE_NAME})
+    ly_create_alias(NAME ${MODULE_NAME}.Tools NAMESPACE Gem TARGETS ${MODULE_NAME})
```

Comment:
Yes, this should have been changed to GEM_NAME. I'm not sure why it's still working, but I'll fix it in the new version.

Reply:
Probably because there's nothing wrong with creating an alias called ".Builders" or ".Tools" - they just won't really have any effect...  in terms of how come its working thats the real mystery, since its not likely its loading at all

Reply:
These are now in the Find_Pyside file of the pyside package.